.github/workflows/helm-chart-release.yaml

```
@@ -10,6 +10,7 @@ on:
 
 jobs:
   release:
+    if: ${{ github.repository == 'kubescape/kubescape' }} # run only in KS org
```

```suggestion
    if: ${{ github.repository == 'kubescape/helm-charts' }} # run only in KS org
```

It will be better if we check only the org name.